Hedgehog.sln

```diff
@@ -7,13 +7,13 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution items", "Solution
 		README.md = README.md
 	EndProjectSection
 EndProject
-Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "Hedgehog.Benchmarks", "tests/Hedgehog.Benchmarks/Hedgehog.Benchmarks.fsproj", "{7E2DE9D9-3831-46D7-B80F-B643E9D53FD3}"
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "Hedgehog.Benchmarks", "tests\Hedgehog.Benchmarks\Hedgehog.Benchmarks.fsproj", "{7E2DE9D9-3831-46D7-B80F-B643E9D53FD3}"
```

Comment:
Are those changes in `Hedgehog.sln` necessary?

Reply:
No, this changes are not necessary. I guess it happened because i used `dotnet sln add` and `dotnet sln remove` for Hedgehog.Fable.Tests. I reverted the changes.